js/scripts/pre-push

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+branch=$(git rev-parse --abbrev-ref HEAD)
+
+# Check that the branch starts with `ui` or `js`
+# and that `npm` is installed
+if [[ "$branch" =~ ^(ui|js) ]] && npm --version 1>/dev/null 2>/dev/null;
```

Comment:
I'd rather do it on everything, it is fast enough - plus, we are doing more-and-more of the combined PRs especially in the jsapi realm.